dask/dataframe/_hyperloglog.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*
+u"""Implementation of HyperLogLog
+
+This implements the HyperLogLog algorithm for cardinality estimation, found
+in
+
+    Philippe Flajolet, Éric Fusy, Olivier Gandouet and Frédéric Meunier.
+        "HyperLogLog: the analysis of a near-optimal cardinality estimation
+        algorithm". 2007 Conference on Analysis of Algorithms. Nice, France
+        (2007)
+
+"""
+from __future__ import division
+
+import numpy as np
+import pandas as pd
+
+def compute_first_bit(a):
+    # Consider making this less memory-hungry
+    #bits = np.bitwise_and.outer(a, 1<<np.arange(31, -1, -1))
+    bits = np.bitwise_and.outer(a, 1<<np.arange(32))
+    bits = bits.cumsum(axis=1).astype(np.bool)
+    return 33 - bits.sum(axis=1)
+
+def terrible_dataframe_hash(df):
```

Comment:
That's true, it's only valid for platforms where multiprocessing works via fork (bsd, linux and osx), since the seed is copied over to each child process. Since windows lacks fork, the seeds are unique per child (and this fails). Do you have a good suggestion for a fix?